src/main/kotlin/pl/put/smartgarden/infra/api/UserController.kt

```diff
@@ -119,4 +129,130 @@ class UserController(val userService: UserService) {
         // TODO
         return UserResourceDto("nazwauzytkownika", "uzytkownik@gmail.com", "123dasads123")
     }
+
+    @GetMapping("/area/{areaId}/measures")
+    @ApiOperation("Get measures for selected area")
+    @ApiResponses(value = [
+        ApiResponse(code = 200, message = "OK"),
+        ApiResponse(code = 400, message = "Bad request"),
+        ApiResponse(code = 404, message = "Not found")
```

Comment:
Annotacje @Api*** nie mają znaczenia z punktu widzenia działania aplikacji. Są jedynie informacją co ma się wyświetlić w swaggerze, a więc czego mogą się spodziewać Wojtek i Weronika odpytując ten endpoint - myślę że musimy jakoś wspólnie ustalić co zwracamy gdy został wysłany zły token, 403 czy 404? Chyba bardziej pasuje 403, przy czym lepszą informacją jesy "Unauthorized" niż "Unactivated". To tylko kosmetyczny szczegół

Reply:
Unauthorized to 401 :P

Reply:
ja bym poszedl w 401 wlasnie

Reply:
W ogóle te annotacje mi sie nie podobają bo cała klasa jest tym zawalona xd

Reply:
ale wiem, ze to dla Wery i Wojtka jest ultra potrzebne

Reply:
ale to chyba oczywiste ze jak rzucimy 401, to ze chodzi o token, ja bym sie tak nie rozdrabnial nad tym swaggerem tbh, a przynajmniej te kody odpowiedzi mysle ze są zbedne

Reply:
Możemy wywalić te kody lub zostawić jak jest i ewentualnie później dopasować - i tak już dużo więcej endpointów nie będzie 🤷‍♂️